pygeo/mphys/mphys_dvgeo.py

```diff
@@ -303,23 +465,28 @@ def nom_addTriangulatedSurfaceConstraint(
         self.add_output(f"{name}_KS", distributed=False, val=0)
         self.add_output(f"{name}_perim", distributed=False, val=0)
 
-    def nom_addRefAxis(self, childIdx=None, **kwargs):
-        # references axes are only needed in FFD-based DVGeo objects
-        if self.geo_type != "ffd":
-            raise RuntimeError(f"Only FFD-based DVGeo objects can use reference axes, not type:{self.geo_type}")
-
-        # we just pass this through
-        if childIdx is None:
-            return self.DVGeo.addRefAxis(**kwargs)
-        else:
-            return self.DVGeo.children[childIdx].addRefAxis(**kwargs)
-
     def nom_setConstraintSurface(
         self, surface, name="default", addToDVGeo=False, DVGeoName="default", surfFormat="point-vector"
     ):
         # constraint needs a triangulated reference surface at initialization
         self.DVCon.setSurface(surface, name=name, addToDVGeo=addToDVGeo, DVGeoName=DVGeoName, surfFormat=surfFormat)
 
+    def nom_writeSurfaceSTL(self, fileName, surfaceName="default", fromDVGeo=None):
```

Comment:
Do we want to duplicate this function here? Since it isn't "MPhys-critical" (ie not a DV or constraint) it might be better to leave this out of the wrapper. It should still be accessible through the runscript anyway just with a slightly longer function handle. Thoughts @anilyil and others?